coopr-server/src/main/java/co/cask/coopr/spec/template/LeaseDuration.java

```diff
@@ -130,4 +190,48 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hashCode(initial, max, step);
   }
+
+  /**
+   * Builder for {@link LeaseDuration}.
```

Comment:
It's not immediately clear how we expect timestamps as strings to be formatted. Add a javadoc so it is clear.